### ISSUE TITLE
[MAS1.4.3] [Visual Requirements-Sign in with Azure account dialog] Luminosity Ratio is less than 4.5:1 for “Don’t have azure account? Sign up, Learn more about registering services” links 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1740]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 for “Don’t have azure account? Sign up, Learn more about registering services” links

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Don’t have azure account? Sign up, learn more about registering services” links

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3.Navigate to the Sign in with your azure account and select it
4.Dialog gest opened
5.Navigate to the “Don’t have azure account? Sign up, Learn more about registering services” links
6.Open 'Colour Contrast Analyser' application.
7.Verify whether Luminosity Ratio is less than 4.5:1 for “Don’t have azure account? Sign up, Learn 
more about registering services” links

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
![MAS1 4 3_Sign in with azure account](https://user-images.githubusercontent.com/53518994/63253757-0d8e0080-c290-11e9-97c9-c4f48a3bad1a.png)
